cmake/ActsConfig.cmake.in

```diff
@@ -67,6 +71,18 @@ if(PluginTGeo IN_LIST Acts_COMPONENTS)
   find_dependency(ROOT @ROOT_VERSION@ CONFIG EXACT)
 endif()
 
+# dependencies that we have built ourselves but cannot be
+# straightforwardly handed to cmake
+if(NOT @ACTS_USE_SYSTEM_BOOST@)
+  add_library(Boost::boost INTERFACE IMPORTED GLOBAL)
+  target_include_directories(Boost::boost INTERFACE "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}")
+endif()
```

Comment:
Stupid question, if this is an interface lib, a downstream project can't use any boost shared libs, can it? Or does it find them because they're installed in the install lib dir?

Reply:
The Boost::boost target here is only for the headers, so that's why I kept it as interface. The shared libraries are installed in the lib/ directory but the targets are not exported, but there's no fundamental reason that they couldn't be so let me knwo if you prefer that we just export everything, not just the headers